roles/swift-account/templates/etc/swift/account-server.conf

```diff
@@ -25,7 +25,10 @@ use = egg:swift#recon
 use = egg:swift#account
 
 [account-replicator]
+concurrency = {{ swift_account.replicators }}
 
 [account-auditor]
+concurrency = {{ swift_account.auditors }}
 
 [account-reaper]
+concurrency = 1
```

Comment:
The variable is specific to account server.. So didn't create variable in common role. .But no harm in creating one in account role, will do that...